app/views/shared/_stats.html.erb

```diff
@@ -0,0 +1,7 @@
+<% @user ||= current_user %>
+<div class="stats">
+  <%= link_to raw(t(".following", count: @user.following.count)),
+    following_user_path(@user) %>
+  <%= link_to raw(t(".followers", count: @user.followers.count)),
```

Comment:
cái raw này sẽ đưa chuỗi về dạng đúng. vì bên trong n đã có html_safe r ạ nhưng n an toàn hơn html_safe :
![image](https://user-images.githubusercontent.com/28645302/28099297-c88b6f0c-66e5-11e7-84d2-81b35073f650.png)